packages/next/src/generators/application/application.spec.ts

```diff
@@ -199,23 +200,7 @@ describe('app', () => {
         "import './global.css';
 
         export const metadata = {
-          title: 'Welcome to ${name}',
-          description: 'Generated by create-nx-workspace',
-        };
-
-        export default function RootLayout({
-          children,
-        }: {
-          children: React.ReactNode;
-        }) {
-          return (
-            <html lang="en">
-              <body>{children}</body>
-            </html>
-          );
-        }
-        "
-      `);
+          title: 'Welcome to `);
```

Comment:
This is part of a group of tests that are skipped via `xdescribe` so the snapshot was never correct in the first place. 

Updated it to be empty so that it can fail if this test is enabled